tensorflow_addons/seq2seq/loss_test.py

```diff
@@ -24,6 +24,37 @@
 from tensorflow_addons.utils import test_utils
 
 
+def get_test_data():
```

Comment:
You could already factorize the data preparation with the other class. Do you plan to do that in the next PR instead?

Reply:
Yeah, I don't want to make the PR too big. Big pull requests are a pain to review.

Reply:
Ok!